extensions/cornerstone/src/commandsModule.js

```diff
@@ -242,9 +245,37 @@ const commandsModule = ({ servicesManager }) => {
     setCornerstoneLayout: () => {
       setCornerstoneLayout();
     },
+    jumpToMeasurement: ({
+      viewports,
+      studyInstanceUid,
+      sopInstanceUid,
+      frameIndex,
+    }) => {
```

Comment:
This is called `jumpToMeasurement`, but it takes `viewports, studyInstanceUid, sopInstanceUid, frameIndex`
It's more of a `jumpToImage`

Reply:
Updated, thanks.